src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import Navigation from "../../components/navigation/Navigation";
+
+function Subreddit() {
+    const [post, setPost] = useState([]);
+    const {id} = useParams();
+    console.log(id);
+
+    useEffect(() => {
+        async function fetchDataSubReddit() {
+            try {
+                const result = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+                console.log(result);
+                setPost(result.data.data);
+            } catch (error) {
+                console.error(error)
+            }
+        }
+
+        fetchDataSubReddit()
+
+    }, [id]);
+
```

Comment:
Hier zul je dan nog een check moeten toevoegen
```suggestion

    if (!post) return null;
```